src/components.js

```diff
@@ -0,0 +1,37 @@
+const text = (elem, text, className) => {
+    let t = document.createElement(`${elem}`);
+    t.textContent = `${text}`;
+    t.classList.add(`${className}`);
+    return t;
+};
+
+const getData = async(place) => {
+    let weather={};
+    
+    await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${place}&lang=en&units=metric&appid=96234945f21abe7a841c8fc0306c59be`)
+        .then(function(response) { 
+            return response.json()
+        })
+        .then(function (data) {
+            weather.country =  data.name
+            weather.sky =  data.weather[0].description
+            weather.icon =  data.weather[0].icon
+            weather.windSpeed =  data.wind.speed
+            weather.windDirection =  data.wind.deg
+            weather.maxTemp =  data.main.temp_max
+            weather.minTemp =  data.main.temp_min
+        })
+        .catch(function(err) {
+            return err
+        });
+    return weather
+}
+
+function appendAll() {
+    const cont = arguments[0];
+    for (let i = 1; i < arguments.length; i++) {
+        cont.appendChild(arguments[i])
+    }
+}
```

Comment:
Use of arrow functions instead of the `function` keyword ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

Reply:
This cannot be done as I'm using the 'arguments variable that does not exist in the arrow function.

Reply:
I have used the arrow function in all the other methods, but this one is a special case.